### PR TITLE
CONSOLE-4124: replace deprecated components in FilterToolbar

### PR DESCRIPTION
**Pull request by rhamilto**
**Note:**  [CONSOLE-4124](https://issues.redhat.com//browse/CONSOLE-4124) will be addressed by 4 PRs, of which this is 1.

After

https://github.com/user-attachments/assets/e7143a74-77a4-4f62-90ff-541ba92ceb91

### CODE REVIEW

File: frontend/public/components/filter-toolbar.tsx [hunk 1]
```diff
@@ -391,24 +392,30 @@ export const FilterToolbar: React.FC<FilterToolbarProps> = ({
...
-                      isGrouped
-                      maxHeight="60vh"
+                      onOpenChange={(open) => setIsOpen(open)}
+                      onSelect={(event: React.MouseEvent | React.ChangeEvent, value: string) => {
+                        onRowFilterSelect(value);
+                      }}
+                      selected={selectedRowFilters}
+                      // maxMenuHeight="60vh" A bug in the current PatternFly version prevents usage
                     >
                       {dropdownItems}
```
Comment (rhamilto) [inline frontend/public/components/filter-toolbar.tsx:417]: Missing `<SelectList>`

**Comment by rhamilto** [comment]
Tech debt so manually adding labels:
/label px-approved
/label docs-approved

/assign @yapei
/assign @sg00dwin

**Comment by sg00dwin** [comment]
@rhamilto Selecting an option by keyboard isn't working.

**Review by sg00dwin — APPROVED** [review]
/lgtm

**Pull request merged**